examples/cpp/get-subgrids.cpp

```diff
@@ -5,11 +5,10 @@
 #include <iomanip>
 #include <ios>
 #include <iostream>
+#include <numeric>
```

Comment:
Is `<numeric>` needed?

Reply:
As always, thanks @cschwan for the detailed comments. I addressed everything in [c5f4f27](https://github.com/NNPDF/pineappl/pull/344/commits/c5f4f27b3ebc71cd79074035dffd92c99e09560f) and [8593c86](https://github.com/NNPDF/pineappl/pull/344/commits/8593c86cc611309c90052604aa824f6ce45e504c).

As the `numeric`, yes, it is needed here because of `std::accumulate`.